test/functional/plugin/lsp/utils_spec.lua

```diff
@@ -265,6 +267,38 @@ describe('vim.lsp.util', function()
 
         eq(56, opts.height)
       end)
+
+      describe('vim.lsp.util.open_floating_preview', function()
```

Comment:
why is this in the `make_floating_popup_options .. with 59 lines above, 20 lines below` describe() block? https://github.com/neovim/neovim/blob/beec377e905baca73e772080c4f276c800ad2a40/test/functional/plugin/lsp/utils_spec.lua#L246